spring-boot-admin-dependencies/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>de.codecentric</groupId>
+        <artifactId>spring-boot-admin</artifactId>
+        <version>1.5.7-SNAPSHOT</version>
```

Comment:
`spring-boot-admin`  brings [full dependency management](https://github.com/codecentric/spring-boot-admin/blob/master/pom.xml#L8) for a particular version of Spring Boot. This BOM should only provide dependency management for the modules that this project handles (as you've indicated in the content  below).